larrecodnn/ImagePatternAlgs/Tensorflow/Modules/RawWaveformDump_module.cc

```diff
@@ -464,6 +464,9 @@ nnet::RawWaveformDump::analyze(art::Event const& evt)
 
     } // loop over SimChannels
 
+    std::vector<raw::ChannelID_t> SelectedChannel;
+    SelectedChannel.clear();
```

Comment:
Searching through an unsorted vector using `std::find` can be inefficient.  In this case, I would recommend using an `std::set`:
```.cpp
std::set<raw::ChannelID_t> selected_channels;
```
Then see below.  Note that it is not necessary to call `clear` on a newly-created vector.